src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/group/RabbitMessageGroupBatchSubscriber.kt

```diff
@@ -43,7 +44,12 @@ class RabbitMessageGroupBatchSubscriber(
     }
 
     override fun extractCountFrom(batch: MessageGroupBatch): Int = batch.groupsCount
-    override fun valueFromBytes(body: ByteArray): List<MessageGroupBatch> = listOf(MessageGroupBatch.parseFrom(body))
+    override fun valueFromBytes(body: ByteArray): List<MessageGroupBatch> {
+        val ins = CodedInputStream.newInstance(body)
+        //TODO: extract the value as a config setting
+        ins.setRecursionLimit(200)
+        return listOf(MessageGroupBatch.parseFrom(ins))
```

Comment:
Implement the same approach in RabbitParsedBatchSubscriber, RabbitRawBatchSubscriber classes